src/commands/lambda/functions/commons.ts

```diff
@@ -191,6 +193,14 @@ export const isMissingDatadogEnvVars = () => isMissingDatadogSiteEnvVar() || isM
 
 export const getAllLambdaFunctionConfigs = async (lambda: Lambda) => getLambdaFunctionConfigsFromRegex(lambda, '.')
 
+// Returns false if not all runtimes are of the same RuntimeType across multiple functions
```

Comment:
I think you could eliminate this comment if you made the name of the function more specific. The function checks that the runtime **types** are uniform -- i.e. python3.8 and python3.9 are considered uniform. Also, I always like to name functions as verbs to make it more clear that they're functions. So, I would suggest the name `checkRuntimeTypesAreUniform()`.